integration/constants.go

```diff
@@ -2,18 +2,18 @@ package integration
 
 // Tracks the start ports handed out to different tests, in a bid to minimise conflicts.
 const (
-	StartPortEth2NetworkTests        = 31000
-	StartPortNodeRunnerTest          = 32000
-	StartPortSimulationGethInMem     = 34000
-	StartPortSimulationInMem         = 35000
-	StartPortSimulationFullNetwork   = 37000
-	StartPortSmartContractTests      = 38000
-	StartPortContractDeployerTest    = 39000
-	StartPortWalletExtensionUnitTest = 40000
-	StartPortFaucetUnitTest          = 41000
-	StartPortFaucetHTTPUnitTest      = 42000
-	StartPortTenscanUnitTest         = 43000
-	StartPortTenGatewayUnitTest      = 44000
+	StartPortEth2NetworkTests        = 10000
```

Comment:
What is the reason we need to change ports? Were there conflicts?

Reply:
Yes, that's right. There are still conflicts in ports, although less now. I've also added a log to tell us which process is occupying the conflicting port.